src/adapters/hasura/altUser.adapter.ts

```diff
@@ -94,7 +94,7 @@ export class ALTHasuraUserService {
     bulkToken: string
   ) {
     const decoded: any = jwt_decode(request.headers.authorization);
-    console.log("decoded", decoded)
+    console.log("decoded", decoded);
```

Comment:
remove console log

Reply:
the decoded console.log has been present in it since the beginning so I don't think it is okay to just remove it as it gives clarity while creating a user in the alt 
p.s. it is used for alt